conf.py

```diff
@@ -13,13 +13,19 @@
 copyright = "%s, %s" % (datetime.date.today().year, author)
 release = '1.0'
 
+# Open Graph configuration
```

Comment:
Might be worth adding a short description in this comment to say that these things will be used to create the preview when linking in other channels etc.

Reply:
Added. I'll also add it to the readme. :)